examples/oneapi/dpc/makefile_lnx

```diff
@@ -86,7 +83,7 @@ else
 endif
 
 ifeq ($(threading),sequential)
-    THREADING_LIB := onedal_sequential
+    $(error Sequential version is not supported. To run sequential version please refer to 'examples/daal/cpp')
```

Comment:
That's wrong message, I've suggested the different one. refer to `examples/oneapi/cpp`

Reply:
done